app/models/User.coffee

```diff
@@ -504,6 +504,7 @@ module.exports = class User extends CocoModel
   showGemsAndXp: -> features?.classroomItems ? false
   showHeroAndInventoryModalsToStudents: -> features?.classroomItems and @isStudent()
   skipHeroSelectOnStudentSignUp: -> features?.classroomItems ? false
+  tryClampHeroHealth: -> features?.classroomItems ? false
```

Comment:
Why is this named `try..`?  That seems to indicate we may not be successful.  Can we rename this to be more clear?